site/_posts/2017-01-23-last-week-in-manageiq-last-will-be-first.html.md

```diff
@@ -0,0 +1,59 @@
+
+## Featured
+
+As we are getting used to splitting repos, deleting so many lines and preparing for Angular 2 in the UI, there were 97 [PRs] merged last week in manageiq core, more than the week before, containing 226 [commits] by 43 [contributors]. You can choose from our list of [open issues], if you would like to contribute. Any help is appreciated.
+
+On Wednesday, we participated in the sprint review meeting as usual, as this was for the end of Sprint 52 for Monday, January 16th. You can find the recorded video in the [ManageIQ channel](https://www.youtube.com/watch?v=eVuhBZ53SP0). By the way, 52 is quite nice number. 52 = 2^2 x 13, and about two weeks ago, we had Friday 13th :)
+
+
+## Improved
+
+It looks like that Ansible Tower objects modeling was the most popular or the [most commented] merged PR with 20 files changed. Some deletions and much more new added lines represent adding service models for new Ansible models and much more. Let's [check this](https://github.com/ManageIQ/manageiq/pull/13427) great work by [James](https://github.com/jameswnl)!
+
+We also got new Alerts to product features last week. [This work](https://github.com/ManageIQ/manageiq/pull/13357) by [Jeff](https://github.com/jeff-phillips-18) includes, of course, adding Alerts to product features, making alerts section identifiers unique and moving them under Monitor, and also the UI part. There were some model changes for alerts, adding alerts API, adding alert statuses in another PRs by another contributors. We'll see how Alerts are goin' in next weeks in ManageIQ.
+
+[Pull request](https://github.com/ManageIQ/manageiq/pull/12863) by [Ken](https://github.com/kevensen) adds more than 700 lines of code in one commit to add container components for service model. As he has written, "OpenShift (and theoretically other container platforms) objects should be treated as first class citizens in the Automate Service Model in ManageIQ. This will enable customized workflows using these objects."
+
+[Simon](https://github.com/isimluk)'s not last contribution to ManageIQ can be seen for example in his [big PR](https://github.com/ManageIQ/manageiq/pull/13419) regarding chargebacks and backporting lots of stuff to euwe. 73 commits in this PR is also a nice number, as it is, in our 21st century, the 21st prime number :)
+
+
+## Fixed
+
+There were, definitely, some bugs and issues fixed last week: 33 in [manageiq repo] and 17 in our favorite [manageiq-ui-classic], including euwe and darga backports.
+
+First, as you know, bin/update had a surprise for all of us representing failing by good_migrations gem. Mr. Migration-repairman, named [Libor](https://github.com/lpichler) came to fix this! His work, containing adding module method for getting class which includes the module, removing dependency on ApplicationRecord and much more, can be seen [here](https://github.com/ManageIQ/manageiq/pull/13450)!
```

Comment:
"adding module method for getting class which includes the module" - possibly "adding a method for getting the class which includes the specified module" - though I'm not sure if the way I am understanding that has changed the intended meaning?